src/styles/global.css

```
@@ -1,6 +1,21 @@
 @import './tokens.css';
 
+@font-face {
+  font-family: 'Madimi One';
+  src: url('@fontsource/madimi-one/files/madimi-one-latin-400-normal.woff2') format('woff2');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'Capriola';
```

Stylelint (17.9.0)

[error] Expected no quotes around "Capriola" (font-family-name-quotes)

(font-family-name-quotes)